constants/modals.ts

```diff
@@ -0,0 +1,26 @@
+type Variant = {
+  icon: string;
+  name: string;
+  imgUrl: string;
+};
+
+type Modals = MODALS.share | MODALS.edit | MODALS.deleteFolder;
```

Comment:
요런 느낌으로 표현해주는게 현재 모달을 띄우는 방식을 생각했을 때 의미적으로 괜찮아보이기도 하네요!
```suggestion
type ModalKeys = MODALS.share | MODALS.edit | MODALS.deleteFolder;
```

Reply:
MODALS라는 enum에 있는 값을 모두 표현해주려면 아래와 같이 써줄수도 있습니다!
```suggestion
type Modals = keyof typeof MODALS;
```